### PR TITLE
pad-windows: Fix a couple of warnings, make codacy happy.

### PR DESCRIPTION
**Pull request by lightningterror**

### CODE REVIEW

File: pcsx2/PAD/Windows/DualShock3.cpp [hunk 1]
```diff
@@ -184,10 +182,8 @@ struct LightState
 struct DS3Command
 {
 	unsigned char id;
-	unsigned char unsure;
```
Comment (CookiePLMonster) [inline pcsx2/PAD/Windows/DualShock3.cpp:187]: Is it not breaking ABI with some HID report or external program? This and the below change can technically change the size and layout of this struct.

Comment (lightningterror) [reply pcsx2/PAD/Windows/DualShock3.cpp:187]: I'll drop the structure changes then, better play it safe.

Comment (CookiePLMonster) [reply pcsx2/PAD/Windows/DualShock3.cpp]: FWIW I **think** `[[maybe_unused]]` is valid to be used even on data fields like that. Maybe that's the best way to move forward?

Comment (lightningterror) [reply pcsx2/PAD/Windows/DualShock3.cpp]: Ah right, forgot about that attribute, let's see if it solves it.

Comment (lightningterror) [reply pcsx2/PAD/Windows/DualShock3.cpp:187]: Seems that it still complains.

**Pull request merged**